src/tests/test_prompt_display.py

```
@@ -0,0 +1,119 @@
+        print(f"\n✅ Strategy generation completed!")
+        print(f"   The LLM prompt and response were displayed above.")
+        
+        return True
+        
+    except Exception as e:
+        print(f"❌ Error during test: {str(e)}")
+        return False
+
+def test_different_operation_types():
+    """Test different operation types for better labeling."""
+    print(f"\n🔬 Testing Different Operation Types")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)